.github/workflows/ci.yml

```diff
@@ -1015,14 +1018,24 @@ jobs:
           sphinx-build -M html ./vtkext/public/doxygen ./vtkext/public/doxygen/build/
           sudo mv vtkext/public/doxygen/build/html _site/doc/libf3d/vtkext_doxygen
 
-      - name: Retrieve web assembly binaries
+      - name: Retrieve f3d package
         uses: actions/download-artifact@v4
         with:
-          name: f3d-wasm
-          path: ./web
+          name: f3d-wasm-package
+          path: ${{github.workspace}}/examples/libf3d/web
+
+      - name: Install f3d package
+        working-directory: ${{github.workspace}}/examples/libf3d/web
+        run: |
+          npm install f3d-*.tgz
+
+      - name: Build F3D web app
+        working-directory: ${{github.workspace}}/examples/libf3d/web
+        run: |
+          npm run build
```

Comment:
```suggestion
        run: npm run build
```

Reply:
why do we need to build it ? Is it not already built ?

Reply:
it's F3D web app build here, not the f3d package.
This job will be integrated in `f3d-website` and removed from here later.